### PR TITLE
feature/OE-185-file-upload-and-training

### PR DESCRIPTION
**Pull request by devsquad-mathews-lima**
## Review

For the submitter, initial self-review:

- [ ] Reviewed the test cases added for completeness and possible points for discussion
- [ ] Check the scope of the changeset - is everything in the diff required for the pull request?

For the pull request reviewer(s), this changeset has been reviewed for:

- [ ] Consistency between the changeset and the goal stated above
- [ ] Usage friction - is the proposed change in usage cumbersome or complicated?
- [ ] Complexity - is the proposed change introducing any unjustified complexity? 
- [ ] Thoroughness of added tests and any missing edge cases

### CODE REVIEW

**Review by Ron-Junior — COMMENTED** [review]
I leave a comment and when you have a response tell me, and after all ok.

File: app/Http/Livewire/Castle/ManageTrainings/Videos.php [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+    public bool $showActions;
+
+    public function mount(Collection $contents, TrainingPageSection $currentSection, bool $showActions = true)
+    {
+        $this->contents        = $contents;
+        $this->currentSection  = $currentSection;
+        $this->selectedContent = new TrainingPageContent();
+        $this->showActions     = $showActions;
+    }
```
[the remark this answers is not in the document]
Comment (devsquad-mathews-lima) [reply app/Http/Livewire/Castle/ManageTrainings/Videos.php:36]: Yeah, you're right, the dependencies are really unnecessary, but the method mount is necessary just to add a default value to the selectedContent variable.

[the remark this answers is not in the document]
Comment (devsquad-mathews-lima) [reply app/Http/Livewire/Castle/ManageTrainings/Videos.php:36]: Yeah, you're right, the dependencies are really unnecessary, but the method mount is necessary just to add a default value to the selectedContent variable.

**Review by Ron-Junior — APPROVED** [review]
